website/templates/log_templates.mako

```diff
@@ -191,6 +191,17 @@ invisible on
   {{/ifnot}}
 </script>
 
+<script type="text/html" id="addon_file_renamed">
+    renamed <span class="overflow">{{ params.source.materialized }}</span>
+  {{#if params.source.materialized.endsWith('/')}}
```

Comment:
We do not use handlebars for log templates--they are Knockout templates with optional support for knockout-punches.